src/core/embeds.py

```diff
@@ -182,7 +186,7 @@ async def callback(self, interaction: discord.Interaction):
                     label="Enter a valid Color",
                     placeholder="Examples: red, yellow, #00ffb3, etc.",
                     required=False,
-                    max_length="7",
+                    max_length=7,
```

Comment:
I don't know how your code work on such mistakes

Reply:
yes it should have been an int but even the str works since dpy handles it internally